polkadot/service/src/config.rs

```diff
@@ -62,3 +63,21 @@ impl Default for Configuration {
 		}
 	}
 }
+
+impl Clone for Configuration {
```

Comment:
Can't that be derived?

Reply:
sadly not since the `transaction_pool::Options` doesn't derive `Clone` and it's in a different repo so can't be fixed in this PR.